### PR TITLE
[BUGS-4199] Document phantomjs workaround and unsupported.

### PR DESCRIPTION
**Pull request by kporras07**
Closes BUGS-4199

## Summary

**[External Libraries](https://pantheon.io/docs/external-libraries)** - Update PhantomJS docs to show workaround and that it is unsupported.

**Release**:
- [X] When ready

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/content/external-libraries.md [hunk 1]
```diff
@@ -74,13 +74,17 @@ To confirm the source of the error, log in to the Drupal Admin and click **Repor
 
 If you encounter this error, remove the offending `quotes` property from the CSS.
 
-## PhantomJS
+## PhantomJS (Unsupported)
 
 In its own words, [PhantomJS](https://github.com/ariya/phantomjs/) is a headless WebKit with JavaScript API. It has fast and native support for various web standards: DOM handling, CSS selector, JSON, Canvas, and SVG.
 
+However, PhantomJS development [has been suspended until further notice](https://github.com/ariya/phantomjs/issues/15344). Pantheon keeps the following binaries but they may be removed in the future.
```
Comment (CdrMarks) [inline source/content/external-libraries.md:81]: How about this phrasing instead:
>However, PhantomJS development [has been suspended until further notice](https://github.com/ariya/phantomjs/issues/15344). Pantheon continues to include the following binaries; they may be removed in the future.

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-6936](https://pr-6936--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `64` | `63` | `-1` |
| Best Practice Score | `79` | `79` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-6936` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/32280/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/32280/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-6936--pantheon-docs.my.pantheonfrontend.website/docs)

File: source/content/external-libraries.md (hunk 1 above)
Comment (jazzsequence) [reply source/content/external-libraries.md:81]: +1 I like @CdrMarks' phrasing.

```suggestion
However, PhantomJS development [has been suspended until further notice](https://github.com/ariya/phantomjs/issues/15344). While Pantheon continues to include the following binaries, they may be removed in the future.
```

File: source/content/external-libraries.md [hunk 2]
```diff
@@ -74,13 +74,17 @@ To confirm the source of the error, log in to the Drupal Admin and click **Repor
...
+
 - PhantomJS (1.7.0) is located at `/srv/bin/phantomjs` on your application container.
 - PhantomJS (2.1.1) is located at `/srv/bin/phantomjs-2.1.1` on your application container.
 
+Recently, PhantomJS started erroring on domains with Let's Encrypt. A known workaround for this is to ignore SSL certs error using the following option: `--ignore-ssl-errors=yes`
```
Comment (jazzsequence) [inline source/content/external-libraries.md:86]: I think this is more grammatically correct: 

```suggestion
Recently, PhantomJS started erroring on domains with Let's Encrypt. A known workaround for this is to ignore SSL certificate errors using the following option: `--ignore-ssl-errors=yes`
```

**Review by CdrMarks — APPROVED** [review]
These changes LGTM.

**Review by joan-ing — APPROVED** [review]
LGTM

**Pull request merged**